src/server/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -248,11 +252,12 @@ void AuctionHouseMgr::SendAuctionCancelledToBidderMail(AuctionEntry* auction, SQ
     Player* bidder = ObjectAccessor::FindConnectedPlayer(bidder_guid);
 
     uint32 bidder_accId = 0;
+
     if (!bidder)
         bidder_accId = ObjectMgr::GetPlayerAccountIdByGUID(bidder_guid);
 
-    if (bidder)
-        bidder->GetSession()->SendAuctionRemovedNotification(auction->Id, auction->itemEntry, item->GetItemRandomPropertyId());
+    if (bidder && item)
+        bidder->GetSession()->SendAuctionClosedNotification(auction, 0.0f, false, item);
```

Comment:
this is definitely wrong, this opcode triggers only these 2 ui errors
ERR_AUCTION_SOLD_S and ERR_AUCTION_EXPIRED_S - these are meant for auction owner